client/src/components/squids/SquidForm.jsx

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable jsx-a11y/no-noninteractive-element-to-interactive-role */
+import React, { useState } from "react"
+
+import { useForm } from "react-hook-form"
+import { useQueryClient, useMutation } from "react-query"
+
+import "../../style/form.pcss"
+import { ApiClient } from "../../backend/ApiClient"
+
+export const SquidForm = () => {
+  const {
+    register,
+    reset,
+    handleSubmit,
+    formState: { errors },
+  } = useForm()
+  const [submitMessage, setSubmitMessage] = useState("")
+
+  const resetForm = () => {
```

Comment:
`react-hook-form` takes a `defaultValues` option and I think that will allow you to set those values and not have to set anything in your `reset` call